shared/src/utils/type-state-selector-map.ts

```diff
@@ -0,0 +1,15 @@
+export const typeSelectorMap = {
+    alarmGroup: 'alarmGroups',
+    client: 'clients',
+    hospital: 'hospitals',
+    mapImage: 'mapImages',
+    material: 'materials',
+    patient: 'patients',
+    personnel: 'personnel',
+    simulatedRegion: 'simulatedRegions',
+    transferPoint: 'transferPoints',
+    vehicle: 'vehicles',
+    viewport: 'viewports',
+} as const;
```

Comment:
In general, I'm not fully convinced that making a distinction between singular and plural is good. It adds more code and doesn't (as far as I can see it) more information or clarification to the programmer.

But if you think this is worth it, then go for it 👍

Reply:
We've discussed this in our team and prefer using the singular.